app/migrations/Version20151209103725.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20151209103725 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_channel DROP FOREIGN KEY FK_16C8119E743BF776');
+        $this->addSql('ALTER TABLE sylius_channel DROP FOREIGN KEY FK_16C8119EECD792C0');
+        $this->addSql('DROP INDEX IDX_16C8119E743BF776 ON sylius_channel');
+        $this->addSql('DROP INDEX IDX_16C8119EECD792C0 ON sylius_channel');
+        $this->addSql('ALTER TABLE sylius_channel DROP default_locale_id, DROP default_currency_id');
```

Comment:
I don't think you changed anything related to channels, this migration is invalid :)